packages/extensions/modelerfour/changelog.md

```diff
@@ -1,103 +1,11 @@
-# Changelog for ModelerFour
+# Change Log - @autorest/modelerfour
```

Comment:
Maybe we can keep the old changelog as `changelog-old.md` or something?  It has some useful details in it.

Reply:
I updated the `CHANGELOG.json` to include the changes from the existing changelog